mountain-climbing/src/main/java/com/example/mountainclimbing/service/ImageService.java

```diff
@@ -17,27 +20,32 @@ public ImageService(ImageRepository imageRepository) {
 		this.imageRepository = imageRepository;
 	}
 
-	public Image createImage(Image image, HttpServletRequest req, HttpServletResponse res) {
+	public ResponseEntity<Image> createImage(Image image, HttpServletRequest req, HttpServletResponse res) {
```

Comment:
Add binding result and check if erros exists and return appropriate response entity and status code
201 - Created
400 - Bad reqyest